portlet/exoadmin/src/main/java/org/exoplatform/organization/webui/component/GroupManagement.java

```diff
@@ -40,30 +41,18 @@
     private static final Logger log = LoggerFactory.getLogger(GroupManagement.class);
 
     public static OrganizationService getOrganizationService() {
-        ExoContainer container = ExoContainerContext.getCurrentContainer();
-        OrganizationService orgService = (OrganizationService) container.getComponentInstanceOfType(OrganizationService.class);
-        return orgService;
+        return ExoContainerContext.getCurrentContainer().getComponentInstanceOfType(OrganizationService.class);
     }
 
     public static UserACL getUserACL() {
-        ExoContainer container = ExoContainerContext.getCurrentContainer();
-        UserACL acl = (UserACL) container.getComponentInstanceOfType(UserACL.class);
-        return acl;
+        return ExoContainerContext.getCurrentContainer().getComponentInstanceOfType(UserACL.class);
     }
 
     public static boolean isMembershipOfGroup(String username, String membership, String groupId) throws Exception {
-        boolean ret = false;
         if (username == null)
             username = org.exoplatform.portal.webui.util.Util.getPortalRequestContext().getRemoteUser();
         OrganizationService orgService = getOrganizationService();
-        Collection groups = orgService.getGroupHandler().resolveGroupByMembership(username, membership);
-        for (Object group : groups) {
-            if (((Group) group).getId().equals(groupId)) {
-                ret = true;
-                break;
-            }
-        }
-        return ret;
+        return orgService.getGroupHandler().resolveGroupByMembership(username, membership).stream().anyMatch(group -> group.getId().equals(groupId));
```

Comment:
Can we use orgService.findMembershipByUserGroupAndType() instead of iterate on each user's group ?
We can do 
```suggestion
                return orgService.getMembershipHandler().findMembershipByUserGroupAndType(username,groupId,membership) !=null;
```

Reply:
OK, Done